api/src/main/java/com/example/api/dto/response/task/feedback/ProfessorFeedbackInfoResponse.java

```diff
@@ -0,0 +1,73 @@
+package com.example.api.dto.response.task.feedback;
+
+import com.example.api.dto.response.task.util.FileResponse;
+import com.example.api.error.exception.EntityNotFoundException;
+import com.example.api.error.exception.MissingProfessorFeedbackAttributeException;
+import com.example.api.model.activity.feedback.ProfessorFeedback;
+import com.example.api.model.activity.result.FileTaskResult;
+import com.example.api.model.activity.task.FileTask;
+import com.example.api.model.user.User;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProfessorFeedbackInfoResponse {
+    private Long feedbackId;
+    private Long fileTaskResultId;
+    private String studentEmail;
+    private Long fileTaskId;
+    private String taskName;
+    private String description;
+    private List<FileResponse> taskFiles;
+    private Double points;
+    private String remarks;
+    private List<FileResponse> feedbackFiles;
+
+    public ProfessorFeedbackInfoResponse(ProfessorFeedback professorFeedback) throws MissingProfessorFeedbackAttributeException, EntityNotFoundException {
```

Comment:
takie rzeczy to raczej w serwisie powinny sie dziac

Reply:
w sensie jakies sprawdznie czy null wywalanie exceptionow itd, ale nie zaglebialem sie jak to robi duzy problem to moze zostac

Reply:
racja, przeniosłem to do serwisu